testDataSetup/neo4j/neo4j.go

```diff
@@ -1,23 +1,72 @@
 package neo4j
 
-import "github.com/ONSdigital/dp-api-tests/config"
+import (
+	"bytes"
+	"fmt"
+	bolt "github.com/johnnadratowski/golang-neo4j-bolt-driver"
+	"html/template"
+)
 
-// Teardown ...
-func Teardown() error {
-  config, err := config.Get()
+const ObservationTestData = "../testDataSetup/neo4j/instance.cypher"
+const HierarchyTestData = "../testDataSetup/neo4j/hierarchy.cypher"
+
+// Datastore used to setup data within neo4j
+type Datastore struct {
+	instance   string
+	testData   string
+	connection bolt.Conn
+}
+
+// CypherTemplate allows cypher queries to be updated with new ID
+type CypherTemplate struct {
+	Instance string
+}
+
+// NewDatastore creates a new datastore for a test
+func NewDatastore(uri, instance, testdata string) *Datastore {
+	driver, err := bolt.NewDriver().OpenNeo(uri)
 	if err != nil {
-		return err
+		panic(err)
```

Comment:
i still think this should return to the initialise file and handle error there with a log and `os.Exit(1)`

Reply:
done